test/api_test.js

```diff
@@ -90,6 +90,11 @@ describe("Api", () => {
       const api = new Api(`http://test/${SPV}`, {bucket: "custom"});
       expect(api.defaultBucket).eql("custom");
     });
+
+    it("should accept a safe option", () => {
+      const api = new Api(`http://test/${SPV}`, {safe: true});
```

Comment:
not really related to what we're doing here, but since we're using `http://test/${SPV}` in different places, should we actually define a `remote_url` const somewhere and use it?